smf/sled-agent/non-gimlet/config.toml

```diff
@@ -31,7 +34,7 @@ zpools = [
 #
 # If empty, this will be equivalent to the first result from:
 # $ dladm show-phys -p -o LINK
-# data_link = "igb0"
+data_link = "igb0"
```

Comment:
Should we leave this commented out in `main`?

Reply:
Yep, that was a whoopsie.